src/modules/apps/settings/set.ts

```diff
@@ -1,11 +1,23 @@
 import { merge, __ } from 'ramda'
 import { createAppsClient } from '../../../api/clients/IOClients/infra/Apps'
 
+const castValue = (value: string) => {
+  let parsedValue: any
+  try {
+    parsedValue = JSON.parse(value)
+  } catch (err) {
```

Comment:
You catch the err and make no moves with this, it's ok ?

Reply:
In this case it's ok, we try to parse a json - if we have an error that means that value isn't a json (check here the exceptions it throws https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse)

Reply:
Awesome!